articles/deciding-between-cloud-spanner-and-microsoft-cosmos-db/index.md

```diff
@@ -0,0 +1,55 @@
+Creating a globally distributed database would traditionally require a lot of time and effort. You had to host the database in an own data center and use your own connections. Today&#39;s advancement in cloud computing and platforms as a service (PaaS) has made it easier to create globally distributed and scalable databases. Particularly, the use of SQL-based databases has become more popular in the recent past. These databases provide you with a customized experience of sorting and processing large amounts of raw data. Many database providers exist, and they work on different nodes of the SQL database engines, including [Microsoft Azure Cosmos DB](https://docs.microsoft.com/en-us/azure/cosmos-db/introduction) and [Google Cloud Spanner](https://cloud.google.com/spanner).
+
+This article will discuss the features of both Cloud Spanner and Microsoft Cosmos DB and highlight their advantages and disadvantages. It will also compare the two to help you better choose the cloud database to use in the future.
+
+### Features of Google Cloud Spanner
+
+Google Spanner is a fully managed, scalable, relational database management service.
+
+The platform can scale millions of nodes across multiple regions. Besides being integrated with access and identity management, it provides features such as [logging and auditing](https://cloud.google.com/spanner/docs/audit-logging).
+
+Google Cloud Spanner&#39;s instance is comparable to a relational database management system ([RDBMS](https://www.codecademy.com/articles/what-is-rdbms-sql)) server. This instance is composed of one or multiple databases, which use the same [compute](https://www.zdnet.com/article/what-is-cloud-computing-everything-you-need-to-know-about-the-cloud/) and [storage](https://searchstorage.techtarget.com/definition/cloud-storage) resources. These resources are allocated during [instance](https://cloud.google.com/spanner/docs/instances) creation. Two main configurations, [Regional and Multi-Regional](https://cloud.google.com/spanner/docs/instances), control the compute and storage resources.
```

Comment:
The last sentence is not clear. 2 main configurations are regional and multi-regional. But how do they control the compute and storage resources? Some more emphasis on how distributed databases come into cost computation would be better.